fix(docs): fix variable name in boot script docs

VOMAkSh opened the pull request:
In async start function on line 260, TodoRepo variable is used but during variable injection in constructor, the variable name used is todoRepo so changed the start function variable name to todoRepo from TodoRepo

dhmlau commented:
@VOMAkSh, thanks for the PR. Could you please take care of the following? Thanks!
- Sign the CLA: https://cla.strongloop.com/agreements/strongloop/loopback-next
- Fix the commit linter error. See the [commit message guideline](https://loopback.io/doc/en/lb4/code-contrib-lb4.html#commit-message-guidelines ). My suggested commit message would be something like:
```
fix(docs): fix variable name in boot script docs
```

VOMAkSh commented:
@dhmlau, thanks for reviewing PR. I have addressed the issues you have asked me. Please, check if there are any further issues associated with my PR.

VOMAkSh commented:
@dhmlau thanks for reviewing PR. I have addressed the issues you have asked me. Please, check if there are any further issues associated with PR.

dhmlau commented:
Thanks @VOMAkSh! Your PR has landed! 🎉
